src/backend/src/prisma/migrations/20240129020615_rr_identifier/migration.sql

```diff
@@ -0,0 +1,26 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[identifier]` on the table `Reimbursement_Request` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- AlterTable
+ALTER TABLE "Reimbursement_Request" ADD COLUMN     "identifier" SERIAL NOT NULL;
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Reimbursement_Request_identifier_key" ON "Reimbursement_Request"("identifier");
+
+-- Assign Identifier to all old Reimbursement Requests based on their dateCreated order
+WITH CTE AS (
+    SELECT 
+        "reimbursementRequestId",
+        ROW_NUMBER() OVER (ORDER BY "dateCreated") AS new_identifier
```

Comment:
works well, good job! One fix tho, it's ordering RRs from newest to oldest instead of vice versa

Reply:
<img width="648" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/bbff9ffd-a05a-4974-9f0f-fcd01abb58ea">